task_2.py

```diff
@@ -0,0 +1,12 @@
+salary = 5000  # Ежемесячная зарплата
+spend = 6000  # Траты за первый месяц
+months = 10  # Количество месяцев, которое планируется протянуть без долгов
+increase = 0.03  # Ежемесячный рост цен
+
+# TODO Рассчитайте подушку безопасности, чтобы протянуть 10 месяцев без долгов
+money_capital = 0
+for months in range(1,11):
```

Comment:
```suggestion
for month in range(months):
```
Диапазон значений можно задавать переменной, которую вы ввели ранее, только необходимо называть временную переменную другим именем